packages/matchbox/src/components/Tag/styles.js

```diff
@@ -0,0 +1,150 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const tagBase = props => `
+  display: inline-flex;
+  align-items: center;
+  white-space: nowrap;
+  padding: 1px ${tokens.spacing_300};
+  padding-right: ${props.hasRemove ? '1px' : tokens.spacing_300};
+  border-radius: ${tokens.borderRadius_pill};
+  line-height: ${tokens.lineHeight_200};
+  font-size: ${tokens.fontSize_200};
+  font-weight: ${tokens.fontWeight_normal};
+`;
+
+// Includes button reset styles
+// Key events (space, enter) inherit the onClick handler through <button>
+export const closeBase = () => `
+  border: none;
+  background: none;
+  padding: 0;
+  display: inline-flex;
+  width: 1.25rem;
+  height: 1.25rem;
+  align-items: center;
+  justify-content: center;
```

Comment:
It would be premature optimization at this point, but it may be worth adding a button reset helper down the road. Just something to think about.